### DISCUSSION TITLE
搜索结果里的广告空位

### DISCUSSION THREAD

**Discussion by mirion-dev** [the1812/Bilibili-Evolved#3862] [功能建议 / Ideas]
![1](https://user-images.githubusercontent.com/51433180/206621886-e5ea0b8e-c330-4191-b791-e3debb275ddf.jpg)

能否直接删除空位元素或者用别的图片/视频填补？

**Comment by mirion-dev** [comment]
刚测试了一下，只有在搜索游戏相关内容时才会出现

**Comment by the1812** [comment]
没啥别的能填充的, 只能屏蔽为空白

**Reply by mirion-dev** [reply]
> 没啥别的能填充的, 只能屏蔽为空白

或许直接删除空白元素更好，末尾空几个要比中间空几个体验好得多（

**Reply by the1812** [reply]
倒是可以像以前的首页一样塞个 `🚫已屏蔽广告`
